bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/English.java

```diff
@@ -214,6 +215,7 @@ public English() {
         yml.addDefault(Messages.FORMATTING_ACTION_BAR_TRACKING, "&fTracking: {team} &f- Distance: {distance}m");
         yml.addDefault(Messages.FORMATTING_TEAM_WINNER_FORMAT, "      {TeamColor}{TeamName} &7- {members}");
         yml.addDefault(Messages.FORMATTING_SOLO_WINNER_FORMAT, "                 {TeamColor}{TeamName} &7- {members}");
+        yml.addDefault(Messages.FORMATTING_EACH_WINNER, "{player}");
```

Comment:
This is double, it also appears on line 285

Reply:
my bad sorry